tools/libSwiftScan/CMakeLists.txt

```diff
@@ -47,11 +47,15 @@ add_link_opts(libSwiftScan)
 add_dependencies(compiler libSwiftScan)
 
 
-if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
-swift_install_in_component(TARGETS libSwiftScan
-  ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT compiler
-  LIBRARY DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT compiler
-  RUNTIME DESTINATION "bin" COMPONENT compiler)
+if(CMAKE_SYSTEM_NAME STREQUAL Windows)
```

Comment:
I wonder if this should use `SWIFT_HOST_SDK_VARIANT` or something?

Reply:
Maybe but it looks like it's already working ok so... different change?

Reply:
You cross-compiled with multiple SDKs?